### ISSUE TITLE
MueLu: Driver fails in rerun mode with stack timers

### ISSUE THREAD

**Issue by jhux2** [trilinos/Trilinos#11522]
## Bug Report
@trilinos/muelu 

### Description
When using the MueLu driver in "rerun" mode, i.e., multiple XML inputs in a single file, and with the option "--stack-timer", the driver fails with the error

```
 Base_Timer:stop Failed timer not running
```

The issue is that the Stack Timer's base timer is stopped after Run1 but never restarted.

### Steps to Reproduce

```
 ./MueLu_Driver.exe --xml=scaling-with-rerun.xml  --stacked-timer
```

**Comment by jhux2** [comment]
Fixed by #11521.

**Comment by jhux2** [comment]
Further experiments show this isn't completely fixed.

**Comment by jhux2** [comment]
Fixed now by #11555.
